metadata-ingestion/src/datahub/utilities/sql_parser.py

```diff
@@ -48,10 +57,12 @@ def __init__(self, sql_query: str) -> None:
         self._parser = MetadataSQLParser(sql_query)
 
     def get_tables(self) -> List[str]:
-        return self._parser.tables
+        result = self._parser.tables
+        # Sort tables to make the list deterministic
```

Comment:
Why is deterministic table ordering required here?

Reply:
There were test in metabase which expected to have the same order always. I wanted to make sure all the test are fine without any modification to make sure I did not break anything.